build/lin.sh

```diff
@@ -280,6 +303,10 @@ sed -i'.bak' "/\[ 'xpm'/{N;N;N;N;d;}" gdk-pixbuf/meson.build
 # Ensure meson can find libjpeg when cross-compiling
 sed -i'.bak' "s/has_header('jpeglib.h')/has_header('jpeglib.h', args: '-I\/target\/include')/g" meson.build
 sed -i'.bak' "s/cc.find_library('jpeg'/dependency('libjpeg'/g" meson.build
+if [ $DARWIN_ARM = true ]; then
```

Comment:
Can we remove the conditional and make this change for everything?

Reply:
We could. I didn't want to impact other platforms with a change that's only really needed because I'm skipping libffi.

I'll hold off making this change until I give fixing libffi a shot.

Reply:
Removed the patch